Dockerfile

```
@@ -0,0 +1,25 @@
+FROM python:3.8-slim as builder
+
+WORKDIR /usr/src/app
+
+RUN pip install poetry
+COPY pyproject.toml poetry.lock src/ README.md ./
+
+RUN poetry export -f requirements.txt > requirements.txt & \
+    poetry build
```

It is better to run same stage both install poetry and use poetry command

```suggestion
COPY pyproject.toml poetry.lock src/ README.md ./

RUN pip install poetry && \
    poetry export -f requirements.txt > requirements.txt && \
    poetry build
```